apps/api/batch_import.py

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
How do we use this command? just run python batch_import.py?

Reply:
yeah. I realised we already have python on the server, and after faffing with bash, it seemed a way I know.

You could probably make a separate php command for this, instead, but I didn't know how to do subprocesses nicely in PHP. 

python batch_import.py

Reply:
Use Process from symfony :)